packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -99,8 +100,9 @@ class _MyHomePageState extends State<MyHomePage> {
           // children horizontally, and tries to be as tall as its parent.
           //
           // Invoke "debug paint" (press "p" in the console where you ran
-          // "flutter run", or select "Toggle Debug Paint" from the Flutter tool
-          // window in IntelliJ) to see the wireframe for each widget.
+          // "flutter run", or select "Toggle Debug Paint" from the
+          // "Flutter Inspector" window in Android Studio) to see the wireframe
```

Comment:
This also works in VSCode, incidentally: it's a command. How about this? 
```suggestion
          // Invoke "debug painting" (press "p" in the console or choose the "Toggle
          // Debug Paint" command in Android Studio or Visual Studio Code)
          // to see the wireframe for each widget.
```

Reply:
Good catch, applied